.changelog/16761.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+code: Prevent Kill Timeout greater than Progress Deadline on Update block
```

Comment:
code or core? It'd be nice if the parameters were detailed in their jobspec form, as it makes it easier for users to grok.